frontend/apps/web/src/pages/index.tsx

```diff
@@ -1,152 +1,188 @@
+              Cell: ({ data }) => {
+                if (data.score === 'good') {
                   return (
-                    <Button
-                      onClick={() => navigate('/repository-details')}
-                      variant="rounded"
-                      size="small"
-                      color="success"
-                    >
-                      Good
-                    </Button>
+                    <Bar>
+                      <Button
+                        onClick={() => navigate(`system/${data.name}`)}
+                        variant="rounded"
+                        size="small"
+                        color="success"
+                      >
+                        Good
+                      </Button>
+                    </Bar>
                   )
                 }
-                if (index === 1) {
+                if (data.score === 'bad') {
                   return (
-                    <Button
-                      onClick={() => navigate('/repository-details')}
-                      variant="rounded"
-                      size="small"
-                      color="error"
-                    >
-                      Bad
-                    </Button>
+                    <Bar>
+                      <Button
+                        onClick={() => navigate(`system/${data.name}`)}
+                        variant="rounded"
+                        size="small"
+                        color="error"
+                      >
+                        Bad
+                      </Button>
+                    </Bar>
+                  )
+                }
+                if (data.score === 'mediocre') {
+                  return (
+                    <Bar>
+                      <Button
+                        onClick={() => navigate(`system/${data.name}`)}
+                        variant="rounded"
+                        size="small"
+                        color="warning"
+                      >
+                        Bad
+                      </Button>
+                    </Bar>
                   )
                 }
-                return (
-                  <Button
-                    onClick={() => navigate('/repository-details')}
-                    variant="rounded"
-                    size="small"
-                    color="warning"
-                  >
-                    Mediocre
-                  </Button>
-                )
               },
             },
           }}
-          data={RepositoryData}
+          data={systems}
         />
       </PageLayout.Section>
     </PageLayout>
   )
 }
 
 export default Dashboard
+
+const Subsystems = styled.div`
+  color: ${Colors.textGray};
+`
+const HeaderWrapper = styled.div`
+  display: flex;
+  justify-content: space-between;
+`
+const Buttons = styled.div`
+  margin-top: auto;
+  margin-bottom: auto;
+
+  button {
+    &:first-child {
+      margin-right: 10px;
+    }
+  }
+`
+const Bar = styled.div`
```

Comment:
Does it work the same when width is just 190px ?

Reply:
It work , but it also works fine without <Bar> so i removed it on this case

Reply:
It's not updated to 190px?

Reply:
Done